sound/soc/sof/intel/intel-audio.c

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
```

Comment:
@ranj063 this file doesn't really have anything intel specific. Care to make it generic?

Reply:
@dbaluta oh yeah after my changes yesterday it really isnt Intel specific. Let me move it out